examples/bri-3/src/bri/workgroup/worksteps/capabilities/updateWorkstep/updateWorkstep.command.handler.ts

```diff
@@ -0,0 +1,19 @@
+import { CommandHandler, ICommandHandler } from "@nestjs/cqrs";
+import { WorkstepAgent } from "../../agents/worksteps.agent";
+import { WorkstepStorageAgent } from "../../persistence/workstepsStorage.agent";
+import { UpdateWorkstepCommand } from "./updateWorkstep.command";
+
+@CommandHandler(UpdateWorkstepCommand)
+export class UpdateWorkstepCommandHandler implements ICommandHandler<UpdateWorkstepCommand> {
+  constructor(private agent: WorkstepAgent, private storageAgent: WorkstepStorageAgent) {}
+
+  async execute(command: UpdateWorkstepCommand) {
+    const workstepToUpdate = await this.agent.fetchUpdateCandidateAndThrowIfUpdateValidationFails(command.id, command.name, command.version, command.status, command.workgroupId, command.securityPolicy, command.privacyPolicy);
+
+    this.agent.updateWorkstep(workstepToUpdate, command.name, command.version, command.status, command.workgroupId, command.securityPolicy, command.privacyPolicy);
+
+    await this.storageAgent.updateWorkstep(workstepToUpdate);
+    
+    return;
```

Comment:
why do we have explicit return on multiple commands?

Reply:
Not needed, probably just a leftover from some testing.

Reply:
removed